ext/storage_sources/gcp_store/gcp_storage_source.cpp

```diff
@@ -326,6 +455,27 @@ gcp_file_size(WT_FILE_SYSTEM *file_system, WT_SESSION *session, const char *name
     return 0;
 }
 
+static int
+gcp_file_read(
+  WT_FILE_HANDLE *file_handle, WT_SESSION *session, wt_off_t offset, size_t len, void *buf)
+{
+    WT_UNUSED(session);
+
+    gcp_file_handle *gcp_fh = reinterpret_cast<gcp_file_handle *>(file_handle);
+    std::string object_key = gcp_fh->name;
```

Comment:
Do we need this `object_key` variable given that it's only used once?

Reply:
`object_key` was used for better readability, I'm happy to remove it if it's a bit redundant :))

Reply:
addressed here: [9faf10f](https://github.com/wiredtiger/wiredtiger/pull/8777/commits/9faf10fe9b57e6c7c29f5909f2cb8c35c5a3ec8e)